### PR TITLE
Store Summary, LLM responses in a table

### PR DESCRIPTION
**Pull request by vivekuppal**
Store all LLM responses in appropriate tables.

Refactor audio player for exceptions, code clarity.
Allow customization of voice response in multiple languages.
Use constants for table names everywhere.
Persist llm response interval across application restarts.
Add unit tests.
Fix broken unit tests for speech recognizer.

### CODE REVIEW

File: app/transcribe/gpt_responder.py [hunk 1]
```diff
@@ -81,6 +84,12 @@ def summarize(self) -> str:
                     collected_messages += message_text
                     # print(f'{message_text}', end="")
 
+            # insert in DB
+            inv_id = appdb().get_invocation_id()
+            engine = appdb().get_engine()
+            summary_obj = appdb().get_object('Summaries')
```
Comment (abhinavuppal1) [inline app/transcribe/gpt_responder.py:90]: use const

File: app/transcribe/run_tests.bat [hunk 2]
```diff
@@ -0,0 +1,2 @@
+SET PYTHONPATH=%cd%;%cd%\..\..
+python -m unittest --verbose .\tests\test_audio_player.py
```
Comment (abhinavuppal1) [inline app/transcribe/run_tests.bat:2]: Should be generic so as new test files are added do not need to modify this file

File: app/transcribe/tests/test_audio_player.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
+"""
```
Comment (abhinavuppal1) [inline app/transcribe/tests/test_audio_player.py:1]: Need to add more tests over time

**Review by abhinavuppal1 — CHANGES_REQUESTED** [review]
Update developer guide as well for unit tests

File: app/transcribe/gpt_responder.py (hunk 1 above)
Comment (vivekuppal) [reply app/transcribe/gpt_responder.py]: Fixed

File: app/transcribe/run_tests.bat (hunk 2 above)
Comment (vivekuppal) [reply app/transcribe/run_tests.bat]: 👍

**Review by abhinavuppal1 — APPROVED** [review]

**Pull request merged**